azure-quantum/.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Do we need to add this file?

Reply:
Having this file improves the SDK developer experience when running the unit tests from VS Code.
It's not really "needed", but nice to have :-)